docs/projects/automate-project-git-source.ipynb

```diff
@@ -0,0 +1,2069 @@
+{
```

Comment:
<p>If, however, your files are already compatibile for running with MLRun, you do not need to clone or extract the files - I think we can add a link that examplain about the relative hanlder</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>